Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Collections/BaseObjectCollection.cs

```diff
@@ -74,6 +75,7 @@ public virtual void UpdateCollection()
             for (int i = 0; i < transform.childCount; i++)
             {
                 Transform child = transform.GetChild(i);
+                Undo.RecordObject(child, "ObjectCollection modify transform");
```

Comment:
This doesn't build. The `UnityEditor` namespace is only present in the Editor itself. You'll need to wrap this and the namespace in `#if UNITY_EDITOR`.

![image](https://user-images.githubusercontent.com/3580640/57724810-79811d00-7640-11e9-93ab-a4c685632ee1.png)